OpenRA.Mods.Common/Traits/Buildings/PlaceBuildingVariants.cs

```diff
@@ -21,6 +21,9 @@ public class PlaceBuildingVariantsInfo : TraitInfo<PlaceBuildingVariants>, Requi
 		[Desc("Variant actors that can be cycled between when placing a structure.")]
 		public readonly string[] Actors = null;
 
+		[Desc("Facings reference of variant actors. Must always have the length of number of variant actors + 1, due to the first element always refer to actor itself")]
+		public readonly WAngle[] Facings = null;
```

Comment:
I found the below example from the PR description very useful for understanding how to populate this correctly, and I'm not sure the current comment alone would've achieved that for me.

```
PlaceBuildingVariants:
		Actors: refinery-nw, refinery-ne, refinery-se
		Facings: 640, 896, 128, 384 ## 640 is the actor itself facing
```

My attempt at wording:
> Facing of the non-variant actor, followed by facings for each variant actor. The length equals the length of Actors + 1.

I might be overthinking it so no need to reword it unless you agree. Just adding this comment in case others have any better suggestions.

Reply:
It is amazing, instead of overthinking. I actually really need a description here to put it simple and clearly

Reply:
Update the comment